### DISCUSSION TITLE
GitHub Classroom Workflow: All jobs have failed

### DISCUSSION THREAD

**Discussion by tomasortiz9598** [dds-jv/foro#217] [Consultas de herramientas] [answered]
Buenas, estoy viendo que al subir el trabajo al repositorio https://github.com/dds-utn/tp-macowins-tomasortiz9598.git falla jacoco con el siguiente mensaje:

![image](https://user-images.githubusercontent.com/24701626/116758925-82af5400-a9e7-11eb-94be-035e3d52127f.png)

El tema es que los tests del paquete Tienda están hechos, incluso cuando corro mvn verify en local jacoco no me tira problemas 

![image](https://user-images.githubusercontent.com/24701626/116758998-a8d4f400-a9e7-11eb-9109-da45bc2d4210.png)

Desde ya, muchas gracias.

Saludos

**Comment by nanderson87** [comment, marked as answer]
Hola Tomas como estás?

Por algun motivo estas usando dos versiones de JUnit? Mi recomendación es que solo uses uno y que para el TP sea Junit5.
Por lo que del pom borres las siguientes dependencias:

```
      <dependency>
          <groupId>junit</groupId>
          <artifactId>junit</artifactId>
          <version>4.11</version>
          <scope>test</scope>
      </dependency>
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>4.11</version>
      <scope>test</scope>
    </dependency>
```

Después de eso vas a tener que solucionar los imports en los test que hacen referencia a JUnit4

```
import org.junit.Assert;
import org.junit.Before;
```

Cualquier cosa avisanos.

**Reply by gastonprieto** [reply]
te sirvio @tomasortiz9598 ?
